tool/Cargo.lock

```diff
@@ -1,7 +1,5 @@
 # This file is automatically @generated by Cargo.
 # It is not intended for manual editing.
-version = 3
```

Comment:
This shouldn't be getting removed: The new toolchain uses the v3 format.

Reply:
@crawfxrd how would I check that? I ran the deps script proof to this.

Reply:
`rustup self update`. Also check `cargo --version` to make sure the correct toolchain is being used (2021-06-12).

Reply:
```bash
$ cargo --version
cargo 1.47.0-nightly (aa6872140 2020-07-23)

$ rustup self update
info: checking for self-updates
info: downloading self-update
  rustup updated - 1.24.3 (from 1.23.1)

$ cargo --version
info: syncing channel updates for 'nightly-2021-06-15-x86_64-unknown-linux-gnu'
info: latest update on 2021-06-15, rust version 1.55.0-nightly (539d7bd39 2021-06-14)
info: downloading component 'cargo'
info: downloading component 'clippy'
info: downloading component 'rust-docs'
info: downloading component 'rust-src'
info: downloading component 'rust-std'
info: downloading component 'rustc'
info: downloading component 'rustfmt'
info: installing component 'cargo'
info: installing component 'clippy'
info: installing component 'rust-docs'
info: installing component 'rust-src'
info: installing component 'rust-std'
info: installing component 'rustc'
info: installing component 'rustfmt'
cargo 1.54.0-nightly (44456677b 2021-06-12)
```